user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Migrating_existing_systems/Migrating_the_general_database_to_a_DMS_Cassandra_cluster.md

```diff
@@ -103,6 +103,9 @@ If your system does not use an indexing database yet or if it already uses a Ope
 
 ### [Running a migration with bespoke Elasticsearch data](#tab/tabid-2)
 
+> [!NOTE]
+> To migrate to a DMS with Cassandra cluster and OpenSearch cluster, it is not possible to use this procedure.
```

Comment:
Can we give any indication of what *is* possible to migrate a system with bespoke Elasticsearch data to an OpenSearch cluster?

Reply:
Afraid not, until we've completed the validation in [DCP222224](https://collaboration.dataminer.services/task/222224) it's uncertain whether there even exists an upgrade path.

If the upgrade path is feasible then the steps would be to first upgrade the Elastic to Opensearch and then perform a regular migration. There is however no DataMiner version which supports this path currently. We're pending on work in Data Core for this.

Is it appropriate in documentation to indicate that a path for this will be provided in the future?

Reply:
How about this?

```suggestion
> To migrate to a DMS with Cassandra cluster and OpenSearch cluster, it is not possible to use this procedure. An alternative procedure will become available for this in the future.
```